roles/pulsar-post-tasks/tasks/main.yml

```diff
@@ -3,6 +3,7 @@
   package:
     name: slurm-drmaa1
     state: present
+  when: install_drmaa is defined and install_drmaa
```

Comment:
adding this will prevent drmaa from being installed on pulsars without `install_drmaa: true` somewhere.  Could be in group_vars/pulsarservers.yml

Reply:
good idea